pennylane_lightning_kokkos/_version.py

```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.30.0-dev1"
+__version__ = "0.30.0-rc0"
```

Comment:
Quick question. Are we merging the change to the version string to the main branch?

Reply:
You are correct - that should've been caught. Thanks.

Reply:
Since we need to build and upload via main/master right now, I decided to keep it frozen to allow RC wheels to be built and uploaded easily for the PennyLane test pipelines, but it will be updated next week